### ISSUE TITLE
expand `BackendProperties` in `PassManagerConfig`

### ISSUE THREAD

**Issue by ajavadia** [Qiskit/qiskit-terra#3626]
As @taalexander astutely pointed out, it makes more sense for the transpiler to accept individual properties rather than a monolithic `BackendProperties`. For example `readout_errors`, `cx_errors`, `gate_durations`, etc, etc. This would be in the same vain as how it currently accepts `basis_gates` and `coupling_map`, rather than `BackendConfiguration`. 

Of course `transpile(circuit, backend)` would still work, but now we would be allowed to override specific properties.

This should be done after #3035. It applies to the `NoiseAdaptiveLayout` pass and the `CrosstalkAdaptiveSchedule` pass.

**Comment by BlakeGerard** [comment]
So essentially, the goal is to allow the direct passing of the parameters in BackendProperties __init__ method to the transpiler?

**Comment by taalexander** [comment]
I'm not positive I still agree with this earlier sentiment as the `Backend` has evolved to be a representation of a target machine for Qiskit (not the transport representation it once was). Perhaps this should be closed now @ajavadia?

**Comment by BlakeGerard** [comment]
Ah ok, I'm just looking around for good first issues.

**Comment by mrvee-qC** [comment]
# Update for Santa:elf:- QGoGP
✅ As mentioned above, there have been significant changes in the code structure as to what `Backend` signifies now! If OP does not have an update on the same, we can safely close this issue! 😄 

Will update comment/add more comments if cause is found/fixed!

> Python version 3.9.7
> qiskit-terra version: 0.19.1

**Comment by jakelishman** [comment]
At the very least, this issue is somewhat superseded by work on `BackendV2`, and we have other trackers for how we want to handle transpilation with respect to those backends (the main one being #7113).  I'll close this in favour of #7113.
